common/models/user.js

```diff
@@ -668,6 +668,45 @@ module.exports = function(User) {
       next();
     });
 
+    // Delete old sessions once email is updated
+    var oldEmail, newEmail, oldId, newId;
+    UserModel.observe('before save', function(ctx, next) {
+      if (ctx.instance) {
```

Comment:
@bajtos I am not sure if that's what you meant : 
replacing the whole `if (ctx.instance)` block by this if statement??
```if (ctx.isNewInstance) return next();```